src/modules/eventHandle.js

```diff
@@ -51,15 +53,27 @@ const eventHandle = (tasks) => {
           task.description = e.target.value;
         }
       });
+      createList(tasks);
       setData(tasks);
     }
   };
 
+  todoList.onchange = (e) => {
+    completed(e, tasks);
+    setData(tasks);
+  };
+
   clear.onclick = () => {
     tasks.splice(0, tasks.length);
     createList(tasks);
     setData(tasks);
   };
+
```

Comment:
- [ ] nice job on changing the status of the task to completed when checked but when I try to clear all completed tasks it removes all tasks please change that function because it doesn't work properly.
**[Hint] You can use the `Array.filter()` method it will be perfect for this situation**